Mock_Data/mock_data_gen.py

```diff
@@ -38,7 +38,9 @@ def simulate(self,start_date,end_date,interval):
             if dates < 5: # weekdays
                 temp = start_date
                 while temp <= start_date + timedelta(hours = 7, minutes = 30): # interval is # of times one wish to scan per hour
-                    cur_price = random.uniform(.98 * cur_price, 1.03 * cur_price) # 2% max change in price since aapl is large stock
+                    cur_price = (1 - (10 - random.randint(0,20))/100) * cur_price # 2% max change in price since aapl is large stock
```

Comment:
You can probably write another method that takes a lower bound, an upper bound, and a price to generate the new price

Reply:
Fixed both issues, added parameters for adjusting lower and upper limit